Make.inc

```diff
@@ -239,6 +239,8 @@ BUILD_LLVM_CLANG := 0
 BUILD_LLDB := 0
 USE_POLLY := 0
 USE_POLLY_OPENMP := 0
+USE_POLLY_ACC := 0
+CUDALIB_INCLUDE_DIR := /usr/local/cuda/include
```

Comment:
Sorry, but I don't really know anything about CUDA (or Polly, for that matter), so I'm not sure how best to address that. Maybe @SimonDanisch could help?